dspace-api/src/main/java/org/dspace/discovery/SearchUtils.java

```diff
@@ -74,34 +77,67 @@ public static void clearCachedSearchService() {
     }
 
     public static DiscoveryConfiguration getDiscoveryConfiguration() {
```

Comment:
if possible please add a javadoc here as it was originally missing. Just to say that it is a shortcut of ... with all null parameter